Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1291,9 +1291,10 @@ def create_test_file(is_nightly, skip_save=False, path_to_pack=''):
             last_commit, second_last_commit = commit_string.split()
             files_string = tools.run_command("git diff --name-status {}...{}".format(second_last_commit, last_commit))
 
-        two_before_ga = AMI_BUILDS.get('TwoBefore-GA', '0').split('-')[0]
+        minimum_server_version = AMI_BUILDS.get('OneBefore-GA', '0').split('-')[0]
```

Comment:
shouldn't this be `GA`? `OneBefore-GA` is 4.5 and that will also cause an endless loop since all content is 5.0+
```suggestion
        minimum_server_version = AMI_BUILDS.get('GA', '0').split('-')[0]
```

Reply:
I think we still run tests on `OneBeforeGA`.

Reply:
![Screen Shot 2020-09-24 at 10 44 00](https://user-images.githubusercontent.com/21142167/94116232-71033e00-fe53-11ea-90c4-9cefbb80152a.png)
seems fine

Reply:
@yaakovi seems to work, are you sure?

Reply:
ok then leave as is.

Reply:
Why OneBeforeGA is 4.5?
GA = 6.0
OneBeforeGA = 5.5
TwoBeforeGA = 5.0
no?

Reply:
@anara123 The AMI names don't follow the actual GA notation. So when we release a new version the AMI with the name `OneBeforeGA` is not actually one before GA, it's just called that.

We have made an effort to fix this going forward since it's confusing.

Reply:
got it